os/arch/arm/src/armv7-m/up_assert.c

```diff
@@ -440,10 +452,38 @@ static void _up_assert(int errorcode)
 }
 
 #ifdef CONFIG_BINMGR_RECOVERY
+static void binary_manager_activate_faultmsg_sender(void)
```

Comment:
This function is not about activating the thread but about waking or unblocking the faultmag sender.